src/app/page.tsx

```diff
@@ -159,9 +160,11 @@ export default function Home() {
 
       <PageSection
         kicker="Learn"
-        title="Use Cases"
-        description="Learn about projects building on and using Filecoin."
-      />
+        title="Filecoin Use Cases"
+        description="Navigate the Filecoin Ecosystem Explorer, a crowd-sourced and open database to showcase projects powering the Filecoin network."
+      >
+        <FeaturedEcosystemProjects />
```

Comment:
Should it have View All button? 
![Screenshot 2024-05-01 at 12 48 24](https://github.com/FilecoinFoundationWeb/filecoin-foundation/assets/50910606/e848c4f4-e25c-47fb-82b0-b4bcd1d458f9)

Reply:
Totally. 🙏🏼